internal/postgres/tablemap_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package postgres
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetSpannerTable(t *testing.T) {
+	conv := MakeConv()
+	basicTests := []struct {
+		name    string // Name of test.
+		pgTable string // PostgreSQL table name to test.
+		error   bool   // Whether an error is expected.
+		changed bool   // Should table name to be changed.
```

Comment:
instead of 'changed' if it was 'expectedTableName' then uniqueness/algorithm test can be covered here.

Reply:
Done.
I was trying to specify the tests in a way that wouldn't be dependent on the specific collision avoidance algorithm. But I'm not sure it is worth doing that. Moreover, the specifics of collision avoidance are API visible in any case, so not unreasonable to test the specifics.